cache/fast_lru_cache.h

```diff
@@ -11,13 +11,22 @@
 #include <memory>
 #include <string>
 
+#include "cache/cache_key.h"
 #include "cache/sharded_cache.h"
 #include "port/lang.h"
 #include "port/malloc.h"
 #include "port/port.h"
 #include "rocksdb/secondary_cache.h"
 #include "util/autovector.h"
 
```

Comment:
Please add a comment like (https://github.com/facebook/rocksdb/blob/main/cache/lru_cache.h#L25-L52) to explain the high level ideas of fast_lru_cache. This can be a TODO for now.